src/main/resources/flyway/migrations/postgresql/V1.0.3__Create_Observation_view.pg.sql

```diff
@@ -0,0 +1,53 @@
+create view f_observation_view as
+SELECT measurement.measurement_id                AS observation_id,
+       measurement.person_id,
+       measurement.measurement_concept_id        AS observation_concept_id,
+       measurement.measurement_date              AS observation_date,
+       measurement.measurement_datetime          AS observation_time,
+       measurement.measurement_type_concept_id   AS observation_type_concept_id,
+       measurement.operator_concept_id           AS observation_operator_concept_id,
+       measurement.value_as_number,
+       NULL :: character varying                 AS value_as_string,
+       measurement.value_as_concept_id,
+       NULL :: integer                           AS qualifier_concept_id,
+       measurement.unit_concept_id,
+       measurement.range_low,
+       measurement.range_high,
+       measurement.provider_id,
+       measurement.visit_occurrence_id,
+       measurement.measurement_source_value      AS source_value,
+       measurement.measurement_source_concept_id AS source_concept_id,
+       measurement.unit_source_value,
+       measurement.value_source_value,
+       NULL :: character varying                 AS qualifier_source_value
+FROM measurement
+UNION ALL
+SELECT (-observation.observation_id)             AS observation_id,
+       observation.person_id,
+       observation.observation_concept_id,
+       observation.observation_date,
+       observation.observation_datetime,
```

Comment:
I started with the script you provided in [this comment](https://github.com/omoponfhir/omoponfhir-main/issues/3#issuecomment-570262581), but I had to change `observation_time` to `observation_datetime`, since there is no field called `observation_time` in the [OMOP CDM v.5.3.1](https://github.com/OHDSI/CommonDataModel/blob/v5.3.1/PostgreSQL/OMOP%20CDM%20postgresql%20ddl.txt#L513-L533).

However, this change is not correct, since the JPA entities are expecting a time string:

```java
@Column(name = "observation_time")
// @Temporal(TemporalType.TIME)
private String time;
```

This probably needs to be fixed before this PR is merged. What is your recommendation @myungchoi?

Reply:
It looks like `v5.0.1` was the last version with `observation_time`. I wrote some PL/pgSQL in cd4c236 to work around this, which will at least allow the view to be created in `v5.3.1`. I expect there will be other more serious issues though, which I'll work through and update you on.

Since nothing in this PR is run automatically, it should still be safe to merge.

Reply:
Thanks for your help. The OMOPonFHIR server is currently running over our research MIMIC server, which was created years ago. So, we are bit behind updating entities to newer version.  I will ask our database team to update our existing database to latest version. The jpabase should then be having correct schema.

Reply:
What is the precise version of the CDM currently supported? Perhaps we could use branches, tags, or releases for specific versions? We could then also disable automatic schema mutation, allowing implementers to use the Flyway migrations in this PR to prepare their databases.